src/databases/models/User.js

```diff
@@ -62,7 +62,10 @@ module.exports = (sequelize, DataTypes) => {
       profile_pic_public_id: {type: DataTypes.STRING, allowNull: true},
       status: {type: DataTypes.BOOLEAN, allowNull: false},
       bio: {type: DataTypes.TEXT, allowNull: true},
-      blocked_by_admin: {type: DataTypes.BOOLEAN, allowNull: true}
+      blocked_by_admin: {type: DataTypes.BOOLEAN, allowNull: true},
+      is_anonymous: {type: DataTypes.BOOLEAN, allowNull: false},
+      combined_user_score: {type: DataTypes.DOUBLE, allowNull: false},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `is_anonymous` and `combined_user_score` fields are set to not allow null values (`allowNull: false`). However, this might cause issues when existing users in the database don't have these fields populated yet. Consider setting `allowNull: true` or providing default values.
```diff
-       is_anonymous: {type: DataTypes.BOOLEAN, allowNull: false},
-       combined_user_score: {type: DataTypes.DOUBLE, allowNull: false},
+       is_anonymous: {type: DataTypes.BOOLEAN, allowNull: true, defaultValue: false},
+       combined_user_score: {type: DataTypes.DOUBLE, allowNull: true, defaultValue: 0},
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->